libs/pages/logs/environment/src/lib/feature/pod-logs-feature/pod-logs-feature.tsx

```diff
@@ -21,6 +22,39 @@ const PodLogs = memo(_PodLogs)
 
 export function PodLogsFeature({ clusterId }: PodLogsFeatureProps) {
   const { organizationId = '', projectId = '', environmentId = '', serviceId = '' } = useParams()
+  const [filter, setFilter] = useState<TableFilterProps[]>([])
+  const [searchParams, setSearchParams] = useSearchParams()
+
+  const POD_NAME_KEY = 'pod_name'
+  useEffect(() => {
```

Comment:
I think it missing one behavior, if `filter` is undefined we need to reset `searchParams`, you can see on the Loom:

https://www.loom.com/share/0d53cdfdc43a4e5c8df9f580937639e2

Reply:
Fixed https://github.com/Qovery/console/pull/1142/commits/9f785d7080eba6040873bbaf99c6e3548480bbed